src/pkg/backup/details/sharepoint.go

```diff
@@ -66,6 +66,12 @@ func (i *SharePointInfo) UpdateParentPath(newLocPath *path.Builder) {
 }
 
 func (i *SharePointInfo) uniqueLocation(baseLoc *path.Builder) (*uniqueLoc, error) {
+	if i.ItemType == SharePointList {
+		loc := NewSharePointLocationIDer(path.ListsCategory.HumanString(), baseLoc.Elements()...)
```

Comment:
this doesn't feel right.  The location path usually denotes the parent container of an item.  Ex: if a file is found at `/root/foo/bar/baz.txt`, then the location would be `/root/foo/bar`.  The category isn't an expected part of the location.

Reply:
@ryanfkeepers that is the reason I guess it takes 'prefixElements' count. So that when this is used as 'locationRef' it [eliminates](https://github.com/alcionai/corso/blob/27ff840be855ccc1af19c0f7d73c538ab099c6dd/src/pkg/backup/details/details.go#L104) the prefix part. 

This has been the norm in other services as well. 

I made changes to the NewSharePointLocationIDer function though, it had a default LibraryCategory added to the path within it.